### PR TITLE
[ADD] Card Atom and Atoms folder

### PR DESCRIPTION
**Pull request by AryRosvall**
Adding card atom and atoms folder.

Resolves issue #42

### CODE REVIEW

File: src/components/atoms/Card.svelte [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<style>
+	.card {
+		display: flex;
+		flex-flow: column nowrap;
+		border-radius: 4px;
```
Comment (helbertoro) [inline src/components/atoms/Card.svelte:5]: maybe you can use the variables `--common-border-radius`

File: src/components/atoms/Card.svelte [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		box-shadow: 2px 2px 8px rgba(0,0,0,0.1);
+		padding: 1em;
+		margin: 8px;
```
Comment (helbertoro) [inline src/components/atoms/Card.svelte:8]: maybe you can use the variables `--common-spacing-unit`

File: src/components/atoms/Card.svelte [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<style>
+	.card {
+		display: flex;
+		flex-flow: column nowrap;
+		border-radius: 4px;
+		box-shadow: 2px 2px 8px rgba(0,0,0,0.1);
```
Comment (helbertoro) [inline src/components/atoms/Card.svelte:6]: maybe you can use the variables `--common-container-shadow`

**Comment by AryRosvall** [comment]
@helbertoro you were right I upload an old version of the component. check the new component please.

**Pull request merged**